example/glue/ArrayletObjectModel.hpp

```diff
@@ -56,6 +56,14 @@ class GC_ArrayletObjectModel
 	{
 		/* No-op */
 	}
+
+#if defined(OMR_ENV_DATA64)
+	MMINLINE void
+	setDataAddrForIndexableObject(omrobjectptr_t arrayPtr, void *address)
```

Comment:
Let's drop Indexable from the name. The class name (instance name too, indexableObjectModel) it's giving the clear context.